app/sprinkles/account/locale/en_US/messages.php

```diff
@@ -134,6 +134,9 @@
             'PAGE'              => 'Choose a new password for your account.',
             'SEND'              => 'Set New Password and Sign In'
         ],
+        'SECURITY'  => [
+            'RESET_REQUIRED'    => 'The password security policy does not allow use of known compromised passwords. Either this feature has been activated after you originally set up your accounty or your password has recently been added to the list.'
```

Comment:
Key name is a little too generic. It is foreseeable that we could add new criteria for mandating a password reset in the future (such as changes to required password complexity and sitewide password resets). Don't have any suggestions right now, but if anything comes to me I'll let you know.

Reply:
Maybe something like `SECURITY.RESET_REQUIRED.COMPROMISED`? Deeper nesting would allow for additional security related reset requirements such as `SECURITY.RESET_REQUIRED.INSUFFICENT_COMPLEXITY`, and `SECURITY.RESET_REQUIRED.EXPIRED` (for mandating required routine password changes) or whatever the individual using the platform wants.

Makes it easier to drop features in without having breaking changes too.

Reply:
Great idea. I added this change in the recent commits.